sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/UcfgSerializerTest.cs

```diff
@@ -59,9 +59,9 @@ void Foo(string a, string b)
 0 [shape=record label=""{BLOCK:0|%0 __concat b,a|x __id %0|TERMINATOR JUMP: 1, 2}""]
 0 -> 1
 0 -> 2
-1 [shape=record label=""{BLOCK:1|%0 C.Bar(string, int) a,CONST|TERMINATOR JUMP: 3}""]
+1 [shape=record label=""{BLOCK:1|%1 C.Bar(string, int) a,CONST|TERMINATOR JUMP: 3}""]
 1 -> 3
-2 [shape=record label=""{BLOCK:2|%0 C.Bar(string, int) b,CONST|TERMINATOR JUMP: 3}""]
+2 [shape=record label=""{BLOCK:2|%2 C.Bar(string, int) b,CONST|TERMINATOR JUMP: 3}""]
```

Comment:
Same here?

Reply:
The temp variables do not start from %0 for each block, the numbering is shared for the whole UCFG.